tsdb/raw.go

```diff
@@ -93,6 +93,7 @@ func (e *RawExecutor) execute(out chan *models.Row) {
 	// Keep looping until all mappers drained.
 	var err error
 	for {
+		timeout := time.After(30 * time.Second)
```

Comment:
we aren't sending out in a channel in this one the same way as aggregate.  It's wrapped in a `limitedRowWriter`.  There really doesn't appear to be a good way to `timeout` in the raw scenario. Should we just skip that for now.  At least we have the `CloseNotifier` in place for raw now which is important to free up resources.  // @benbjohnson